jobqueue/src/main/java/com/path/android/jobqueue/network/NetworkUtilImpl.java

```diff
@@ -36,7 +38,16 @@ public boolean isConnected(Context context) {
             return false;
         }
 
+        int permState = context.checkCallingOrSelfPermission(Manifest.permission.ACCESS_NETWORK_STATE);
+        if (permState != PackageManager.PERMISSION_GRANTED) {
```

Comment:
hmm ok, if you've received the exception.
Can you recreate the PR on top of master so i can merge it ? right now it conflicts.